### ISSUE TITLE
Plotly scala 

### ISSUE THREAD

**Issue by Armadik** [jupyter-incubator/sparkmagic#787]
Good afternoon
I can't figure out how to draw graphs using spark scala kernels. The functionality works in pyspark. I would like to get something like this https://plotly.com/scala/user-guide/. Is it possible?

**Comment by Strunevskiy** [comment]
I was wondering what your approach is for plotly with pyspark ?

**Comment by Strunevskiy** [comment]
what did you for for pyspark ? I also have a solution for pyspark but what to compare to others.

**Comment by Armadik** [comment]
@Strunevskiy 
I didn't understand the question a bit. I was interested in the possibility of plotting scala graphs in jupiter notebooks. And for working with graphs through pyspark there should be no problems, there is plotly and so on...

**Comment by Strunevskiy** [comment]
Yes, in pyspark you can install plotly, but how do you bring visualization to notebook ?

**Comment by pancodia** [comment]
In pyspark kernel, I use [`%matplot plt`](https://stackoverflow.com/a/60969284/1933651)
